tests/upload_files.rs

```diff
@@ -275,3 +278,177 @@ fn set_media_type(
 
     Ok(())
 }
+
+#[rstest]
+#[case(server(&["-u"]))]
+#[case(server(&["-u", "-o", "error"]))]
+#[case(server(&["-u", "--on-duplicate-files", "error"]))]
+fn uploading_duplicate_file_is_prevented(#[case] server: TestServer) -> Result<(), Error> {
+    let test_file_name = "duplicate test file.txt";
+    let test_file_contents = "Test File Contents";
+
+    // create the file
+    let test_file_path = write_file_contents(
+        server.path().to_path_buf(),
+        test_file_name,
+        test_file_contents,
+    );
+
+    // Before uploading, make sure the file is there.
+    let body = reqwest::blocking::get(server.url())?.error_for_status()?;
+    let parsed = Document::from_read(body)?;
+    assert!(parsed.find(Text).any(|x| x.text() == test_file_name));
```

Comment:
I think you can get rid of all of this in every test because you can use `fixtures::FILES` and try to overwrite them. They already exist and this should allow you to cut the tests by a lot. We use `fixtures::FILES` in many places in the test suite.